Prevent truncation of stdout/stderr on open(O_TRUNC) in cmake.

--- src/kernel/sel4/config.cmake
+++ src/kernel/sel4/config.cmake
@@ -168,9 +168,6 @@ if(DEFINED KernelDTSList AND (NOT "${KernelDTSList}" STREQUAL ""))
                 "${device_dest}" --hardware-config "${config_file}" --hardware-schema
                 "${config_schema}" --yaml --yaml-out "${platform_yaml}" --arch "${KernelArch}"
                 --addrspace-max "${KernelPaddrUserTop}"
-            INPUT_FILE /dev/stdin
-            OUTPUT_FILE /dev/stdout
-            ERROR_FILE /dev/stderr
             RESULT_VARIABLE error
         )
         if(error)
